devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -82,13 +89,22 @@ private getIkeaOnOffSwitchNames() {
 	]
 }
 
+private getIkeaOpenCloseRemoteNames() {
+	[
+			"Up", // Up button
+			"Down" // Down button
```

Comment:
nit: this is a bit too tabbed for me, personally

Reply:
Fixed